packages/labs/task/README.md

```diff
@@ -0,0 +1,66 @@
+# @lit-labs/task
+
+A controller for Lit that renders asynchronous tasks.
+
+## Overview
+
+Often a Lit element needs to request, process, and render remote data, for
+example when querying a REST API for data to be displayed. The `AsyncTask`
+controller provides a simple pattern for encapsulating this behavior in an
+easily reusable way. The controller integrates with a host Lit element. The
+user provides a task function and a dependencies function. Whenever the element
+updates, the dependencies are checked and if any have changed, the task is
+initiated. The controller requests an update of the element whenever the task
+status changes. Task status is provided via the `TaskStatus` object which has
+values for `INITIAL`, `PENDING`, `COMPLETE`, and `ERROR`. The task result is
+available via its `value` property, or via the `error` property when an error
+occurs. The task `render` method may also be used to easily render different
+task states. It accepts an object which optionally can implement methods for
+`initial`, `pending`, `complete(value)`, and `error(error)`. These methods
+typically return a Lit `TemplateResult` to render
+
+## Installation
+
+From inside your project folder, run:
+
+```bash
+$ npm install @lit-labs/task
+```
+
+## Usage
+
+Here's an example:
+
+```ts
+import {AsyncTask, TaskStatus} from '@lit-labs/task';
+// ...
+
+class MyElement extends LitElement {
+
+  @state()
+  private _userId: number;
+
+  private _apiTask = new AsyncTask(
+      this,
```

Comment:
(Commenting here since the file is unchanged): Just noting that `AsyncTask` contructor's first argument is typed as `ReactiveElement` which could/should now be `ReactiveElementHost`:

https://github.com/Polymer/lit-html/blob/f547ac3f8ef8ebc252b8aa6b1bd1cb60c014df2f/packages/labs/task/src/async-task.ts#L96-L100

Reply:
Done.